File: app/src/main/java/com/example/app_quizz/HelloActivity.java
package com.example.app_quizz;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

public class HelloActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_hello);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent i = new Intent(HelloActivity.this,MainActivity.class);
                startActivity(i);
                finish();
            }
        },3000);
    }
}

File: app/src/main/java/com/example/app_quizz/HelloActivity.kt
package com.example.app_quizz

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.app_quizz.R
import android.content.Intent
import android.os.Handler
import com.example.app_quizz.MainActivity

class HelloActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_hello)
        Handler().postDelayed({
            val i = Intent(this@HelloActivity, MainActivity::class.java)
            startActivity(i)
            finish()
        }, 3000)
    }
}